eth/rlp/blocks.py

```diff
@@ -29,4 +29,5 @@ def __repr__(self) -> str:
         return f'<{self.__class__.__name__}(#{str(self)})>'
 
     def __str__(self) -> str:
-        return f"Block #{self.number}"
+        hash_start = self.hash[:3].hex()
+        return f"Block #{self.number}-0x{hash_start}..."
```

Comment:
Did you intentionally not use `humanize_hash` from `eth_utils` here (which produces hashs of the format `eb58..84cd`)?

Reply:
I guess it basically comes down to not liking that format. I guess I can get over it, and consistency is better than my personal stylistic taste. :)